### PR TITLE
Fix filter vulnerabilities

### PR DESCRIPTION
**Pull request by nickpell**
checks that column names belong to the table and uses bind parameters

@snopoke @czue @NoahCarnahan

### CODE REVIEW

**Comment by nickpell** [comment]
I think travis is using a different version of sqlalchemy than on my machine

**Comment by czue** [comment]
looks great to me but @snopoke has more context

File: tests/test_filters.py [hunk 1]
```diff
@@ -1,43 +1,63 @@
...
+        cls.mock_table = MockTable()
+        setattr(cls.mock_table, 'c', MockColumns())
+        setattr(cls.mock_table.c, cls.column_name, Column(cls.column_name, type_=String))
+
     def test_raw(self):
         a = RAW('fancy stuff')
-        self.assertEqual(a.build_expression(), 'fancy stuff')
+        self.assertEqual(str(a.build_expression(self.mock_table)), 'fancy stuff')
 
+    # todo - check how this changes the functionality
```
Comment (snopoke) [inline tests/test_filters.py]: Is this todo still valid?

**Comment by snopoke** [comment]
Looks great! Just question re todo.

**Comment by nickpell** [comment]
@snopoke Thanks, todo removed.

**Comment by nickpell** [comment]
:white_check_mark:

**Comment by NoahCarnahan** [comment]
nice!

**Pull request merged**